buildSrc/src/main/kotlin/PublicationInfoExtension.kt

```diff
@@ -1,19 +1,24 @@
 import org.gradle.api.Project
 import org.gradle.api.model.ObjectFactory
 import org.gradle.api.provider.Property
+import org.gradle.api.publish.Publication
 import org.gradle.api.publish.maven.MavenPom
 import org.gradle.api.publish.maven.MavenPublication
 
 open class PublicationInfoExtension(objects: ObjectFactory, val project: Project) {
```

Comment:
Could you add some notes on what this class is about? Link to infos on the role of publications?